tests/nomad/test_metainfo_schema.py

```diff
@@ -0,0 +1,121 @@
+"""This is a code that performs several tests on nexus tool"""
+
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+
+try:
+    from nomad.metainfo import Section
+except ImportError:
+    pytest.skip("nomad not installed", allow_module_level=True)
+
+from typing import Any
+
+from pynxtools.nomad.schema import nexus_metainfo_package
+from pynxtools.nomad.utils import __REPLACEMENT_FOR_NX
+
+
+@pytest.mark.parametrize(
+    "path,value",
+    [
+        pytest.param("name", "nexus"),
+        pytest.param("NXobject.name", "NXobject"),
+        pytest.param(f"{__REPLACEMENT_FOR_NX}entry.nx_kind", "group"),
```

Comment:
Note that these has already been modified to
        pytest.param(rename_nx_to_nomad("NXentry") + ".nx_kind", "group"),

Reply:
That's okay, I can rebase this branch. Btw, I intentionally renamed the function to `remove_nx_for_nomad` because this is what it does (we are not renaming anymore as was the original plan).